example-meal-order/src/app/extensions/package.json

```diff
@@ -6,5 +6,18 @@
   "dependencies": {
     "@hubspot/ui-extensions": "latest",
     "react": "^18.2.0"
+  },
+  "scripts": {
+    "lint": "eslint . --ext .tsx,.ts",
+    "lint:fix": "eslint . --ext .tsx,.ts --fix",
+    "check:types": "tsc --noEmit --jsx react-jsx --esModuleInterop --lib WebWorker --lib esnext --skipLibCheck OrderMealExtension.tsx",
+    "check": "npm run lint && npm run check:types"
+  },
```

Comment:
Did you play around with the install flow for all the changes here or mostly copy-paste from the other example?

Reply:
By that I mean, reviewing and installing the deps yourself and running the scripts to make sure everything about the output makes sense and worked as expected, and that the deps are actually needed. It's important to verify everything in here, be critical!